src/lamplib/src/genny/curator.py

```diff
@@ -134,6 +134,28 @@ def translate(
     subprocess.run(args, check=True)
 
 
+# Calculate rollups from all the FTDC outputs that are not from canaries.
+def calculate_rollups(output_dir: str, workspace_root: str, genny_repo_root: str) -> None:
+    curator = _find_curator(workspace_root, genny_repo_root)
+    if not curator:
+        raise OSError("Could not find Curator.")
+    for root, dirs, files in os.walk(output_dir):
+        if "internal" not in root:
+            for file in files:
+                if file.endswith(".ftdc") and "canary" not in file:
+                    ftdc_file_name = os.path.join(root, file)
+                    rollup_file_name = ftdc_file_name.replace(".ftdc", ".json")
```

Comment:
Would you mind adding some logging around this that includes the file being processed and the file that's being output?

Reply:
Good point, added